src/libprojectM/TimeKeeper.cpp

```diff
@@ -105,7 +106,12 @@ double TimeKeeper::PresetTimeA()
 
 double TimeKeeper::sampledPresetDuration()
 {
-    std::normal_distribution<double> gaussianDistribution{m_presetDuration, m_easterEgg};
+    if (m_easterEgg < 0.001)
+    {
+        m_easterEgg = 1.0;
```

Comment:
It might make more sense to say something like `m_easterEgg = max(m_easterEgg, 0.001)`. Having it snap to 1.0 only if it gets very small seems unexpected to me.

Reply:
If it's always properly initialized, then I agree.

Setting it to 1 here will make sure the proper default value is used if someone uses 0 as the default value in an external app integration and passes zero in after construction, e.g. in existing apps made by someone who didn't know about this limitation. Besides that, this "easter egg" isn't really noticeable in any case.

Reply:
Another option would be to just return m_presetDuration if m_easterEgg is zero, since that's what the normal distribution would theoretically do with stddev = 0. No action needed though since there are bigger fish to fry.